components/media_player/ui/CardHome.qml

```diff
@@ -309,6 +315,41 @@ Rectangle {
         }
     }
 
+    Item {
+        id: stopButton
+        width: 120; height: 120
+
+        anchors { horizontalCenter:prevButton.right; bottom: parent.bottom; bottomMargin: 80 }
+
+
+        Text {
+            color: Style.color.text
+            text: Style.icon.stop
+            renderType: Text.NativeRendering
+            width: 85; height: 85
+            verticalAlignment: Text.AlignVCenter; horizontalAlignment: Text.AlignHCenter
+            font {family: "icons"; pixelSize: 80 }
+            anchors.centerIn: parent
+            opacity: obj.state === 3 ? 1 : 0
+
+            Behavior on opacity {
+                NumberAnimation { duration: 100; easing.type: Easing.OutExpo }
+            }
+        }
+
+        MouseArea {
+            anchors.fill: parent
+
+            onClicked: {
+                Haptic.playEffect(Haptic.Click);
+                if (obj.state === 3) {
+                    obj.stop();
+                } else {
+
+                }
```

Comment:
This can be removed, nothing is happening here.

Reply:
Rfmoved